examples/.env

```diff
@@ -1,2 +1,6 @@
 LOGURU_LEVEL="INFO"
+RANK=0
```

Comment:
Presently example will just run serial, I checked it with multiple GPUs with mpirun manually. So spoof a  parallel environment here

Reply:
Can we also check manually with `torchrun`?

Reply:
Can you elaborate? I'm not trying add additional checks inside the script that would not be usually done so we dont mislead readers.

Reply:
Just for our own sanity do we know that this example runs with `mpirun` and `torchrun`? I have just seen occasional different behavior with the distributed manager and I would like to ensure that it would run under both conditions.

Reply:
Yep! I checked mpi and torch run on the DGX box. Runs the same.

Added torch run command too and a comment about the .env file.